python/py-pytorch/Portfile

```diff
@@ -183,6 +189,24 @@ if {${name} ne ${subport}} {
     destroot.dir    ${worksrcpath}
     destroot.target install
 
+    if {${python.pep517}} {
+        build.cmd-append \
+                    --skip-dependency-check
+
+        pre-destroot {
+            destroot.args-append \
+                    [glob ${workpath}/torch-*.whl]
+        }
+        destroot.cmd    pip-${python.branch}
+        destroot.target install
+        destroot.args          \
+            --ignore-installed \
+            --no-cache-dir     \
+            --no-dependencies  \
+            --root ${destroot}
+        destroot.post_args
+    }
```

Comment:
this doesn't look right ; why can we not use the "normal" PEP517 build tools here instead of doing `pip` stuff in the destroot phase?